### PR TITLE
Improve the logging tutorial by clarifying DI vs. exlicit logging pipeline

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

**Review by samsp-msft — APPROVED** [review]

**Review by tarekgh — APPROVED** [review]
LGTM.

**Review by CodeBlanch — APPROVED** [review]

**Review by noahfalk — COMMENTED** [review]
LGTM

File: docs/logs/getting-started-aspnetcore/README.md [hunk 1]
```diff
@@ -125,6 +125,18 @@ generic `ILogger<T>` is not an option, `app.Logger` is used instead:
 app.Logger.StartingApp();
 ```
 
+> **Note**
+> There are cases where logging is needed before the [dependency injection
+(DI)](https://learn.microsoft.com/dotnet/core/extensions/dependency-injection)
+logging pipeline is available (e.g. before `builder.Build()`) or after the DI
+logging pipeline is disposed (e.g. after `app.Run()`). The common practice is to
+use a separate logging pipeline by creating a `LoggerFactory` instance.
```
Comment (cijothomas) [inline docs/logs/getting-started-aspnetcore/README.md:133]: @reyang Do you expect to push this to asp.net core logging docs itself, and put a link from here?

Comment (reyang) [reply docs/logs/getting-started-aspnetcore/README.md:133]: Yes, it'll be multiple steps to move the needles.

**Pull request merged**